eslint.config.js

```diff
@@ -0,0 +1,17 @@
+import importPlugin from "eslint-plugin-import";
+import prettier from "eslint-plugin-prettier/recommended";
+
+export default [
+  importPlugin.flatConfigs.recommended,
+  {
+    files: ["src/**/*.{js,mjs}"],
+    languageOptions: {
+      ecmaVersion: "latest",
+      sourceType: "module",
+    },
```

Comment:
This can be omitted, the defaults have the exact same behavior. We also do this in `file-selector`.

```suggestion
```

Reply:
I removed the entire rule as it doesn't seem necessary.